ts/src/flexible-event/privacy.test.ts

```diff
@@ -152,7 +177,7 @@ test('computeConfigData', async (t) => {
       ),
       {
         numStates: 3,
-        infoGain: 1.584926511508231,
+        infoGain: 1.5849265115082316,
```

Comment:
Just curious -- why has this changed?

Reply:
it's probably precision loss as we've changed the formula from `p` to `1 - (1-p)`, mathematically they should be the same, but since we are using float numbers, there can be tiny difference.